src/main/java/nextstep/fp/Lambda.java

```diff
@@ -53,4 +53,14 @@ public static int sumAllOverThree(List<Integer> numbers) {
         }
         return total;
     }
+
+    public static int sumAll(List<Integer> numbers, Conditional c) {
+        int total = 0;
+        for (int number : numbers) {
+            if (c.test(number)) {
+                total += number;
+            }
+        }
+        return total;
```

Comment:
```suggestion
        return numbers.stream()
                .filter(c::test)
                .mapToInt(number -> number)
                .sum();
```
로 stream 표현을 할 수 있겠네요!